server/app/views/v1/volumes/_volume.json.jbuilder

```diff
@@ -5,3 +5,10 @@ json.name volume.name
 json.scope volume.scope
 json.driver volume.driver
 json.driver_opts volume.driver_opts
+json.instances volume.volume_instances.each do |v|
+    json.node v.host_node.name
+    json.name v.name
+end
+json.services volume.services.each do |s|
```

Comment:
Should we add `.includes(:some_models)` here so that it does not need to query related objects on each iteration?